eng/common/templates/job/build-from-source.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+  jobNamePrefix: 'sourcebuild'    # Override for prefix for job names
+  managedOnly: false              # Specifies whether source is managed only. If so, one build platform is sufficient
+  portableOnly: false             # If true, builds one leg with portable flag enabled
+  prebuiltCheck: true             # Specifies whether to enforce prebuilt checking and fail if new prebuilts are introduced
+
+jobs:
+- template: /eng/common/templates/jobs/jobs.yml
+  parameters:
+    enablePublishUsingPipelines: true
+    enablePublishBuildArtifacts: true
+    enablePublishBuildAssets: true
+    artifacts:
+      publish:
+        artifacts: true
+        manifests: true
+    jobs:
+      # Only run fedora32 when either managedOnly or portableOnly is true.  Otherwise, run all legs.
```

Comment:
Why?

This also doesn't look correct? (I can change how I'm parsing it to make it correct but it's not how I read it normally...)

Reply:
Fixed